subjects/devops/file-checker/README.md

```diff
@@ -0,0 +1,54 @@
+## file-checker
+
+### Instructions
+
+In this exercise you will make a script `file-checker.sh` that will check the status of a given file using if statements.
+
+Your script should make the following checks:
+
+- File exists.
+
+- File is readable.
+
+- File is writable.
+
+- File is executable.
+
+It should output the status as in the example in the usage.
+
+You should also handle what to do when no file is provided. The use of `test` command is not allowed for this exercise.
+
+### Usage
+
+```console
+$ ./file-checker.sh file.txt
```

Comment:
Thanks, done.

Reply:
what i meant here was that you can add an ls -l here so that the student can see the perms of that file
I don't see the example so i assume that i didn't say it clear enough

Reply:
I did add it, it's on line 29. Do you mean that you want it before?
![image](https://user-images.githubusercontent.com/18433970/212744091-f62084b7-2a64-4d3f-b941-e01504b86a7b.png)

Reply:
must have missed it sorry